config/packages/security.yaml

```diff
@@ -41,5 +52,6 @@ security:
      - { path: ^/api/users/register, roles: PUBLIC_ACCESS }
      - { path: ^api/users, roles: ROLE_ADMIN }
      - { path: ^api/admin, roles: ROLE_ADMIN }
+     - { path: ^api/admin/dashboard, roles: ROLE_ADMIN }
```

Comment:
path-ul rutelor pe care le-ai adaugat nu e /admin/*?, cu prefixul api? Daca da, as recomanda sa nu lasi si /api/ acolo pentru ca nu e partea de api asta, daca nu, atunci te rog schimba aici. Poti adauga oricum ceva pentru /admin general si pentru /admin/login sa pui inca o linie deasupra cu role PUBLIC_ACCESS. In felul asta orice ruta /admin va fi protejata, doar pe /admin/login o sa poti intra la liber